src/applications/medical-copays/components/HowToPay.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import Telephone, {
+  PATTERNS,
+} from '@department-of-veterans-affairs/component-library/Telephone';
+
+export const HowToPay = () => (
+  <>
+    <h2>How do I pay my VA copay bill?</h2>
+    <h5>You can pay your bill in any of these 4 ways:</h5>
```

Comment:
Probably need to swap out the h5 to a h3 and slap a looks like h5 on there. Would love to see our a11y check go through the firs time if possible 🙏🏻.

Reply:
yep agree would be nice to get it on the first pass ill update it tks!